slice.go

```diff
@@ -31,6 +31,15 @@ func SliceContains[T any](slice []T, filter func(t T) bool) bool {
 	return false
 }
 
+func SliceEvery[T any](slice []T, filter func(t T) bool) bool {
```

Comment:
```suggestion
func All[T any](slice []T, filter func(t T) bool) bool {
```

And also `Any()`

Reply:
Wouldn't `Any` be the same as `SliceContains`?

Reply:
Hmm, yeah. Nevermind.